website/docs/troubleshooting/troubleshooting-podman.md

```diff
@@ -1,120 +1,157 @@
+- Install from Podman Desktop.
+- Podman installer.
+- Operating system specific installer: Brew, Chocolatey, Scoop, Winget.
+- Installer for restricted environment.
 
-#### Check connection
+Podman Desktop might fail to detect your Podman installation.
 
-Check at least a Podman machine is running on Windows & macOS:
+#### Solution
 
-```bash
-podman machine list
-```
+Try following steps to verify your Podman installation.
+After each step, quit and restart Podman Desktop to verify Podman Desktop can detect your Podman installation.
 
-And check a connection can be made with the CLI
-
-```sh
-$ podman run quay.io/podman/hello
-!... Hello Podman World ...!
-
-         .--"--.
-       / -     - \
-      / (O)   (O) \
-   ~~~| -=(,Y,)=- |
-    .---. /`  \   |~~
- ~/  o  o \~~~~.----. ~~
-  | =(X)= |~  / (O (O) \
-   ~~~~~~~  ~| =(Y_)=-  |
-  ~~~~    ~~~|   U      |~~
-
-Project:   https://github.com/containers/podman
-Website:   https://podman.io
-Documents: https://docs.podman.io
-Twitter:   @Podman_io
-```
+1. In a terminal, verify you can access the Podman CLI, and verify the version.
+
+   ```shell-session
+   $ podman version
+   ```
+
+1. Update Podman to the latest stable version by using your installation method.
+1. Search for errors in the installation logs (if your installation method is providing logs).
+1. Reinstall Podman with the same installation method.
+1. Reinstall Podman with the Podman Desktop installer.
+1. Reinstall Podman with the Podman installer.
+1. Reinstall Podman with another method.
 
-### Unable to see information about active containers
+## Podman Desktop fails creating a Podman machine
 
 #### Issue
 
-In this scenario, the screen might be displaying "No Containers" as shown below despite active containers runnning in the background.
+Podman Desktop might fail creating a Podman machine.
+
+#### Workaround
+
+1. In a terminal, create the Podman machine with the Podman CLI:
+
+   ```shell-session
+   $ podman machine init
+   ```
+
+1. If the creation fails, read carefully the logs to continue troubleshooting.
+
+## Podman Desktop fails starting a Podman machine
+
+#### Issue
+
+Podman Desktop might fail starting a Podman machine.
+
+#### Workaround
+
+1. In a terminal, start the Podman machine with the Podman CLI:
+
+   ```shell-session
+   $ podman machine start
+   ```
+
+1. If the operation fails, read carefully the logs to continue troubleshooting.
+
+## Podman Desktop fails listing images or containers
+
+Podman Destkop might fail listing images or container.
+
+#### Prerequisites
+
+- Podman 4.1.0 or greater. Podman Desktop requires the Podman machine to expose the socket on the host on macOS, and on a named pipe on Windows.
+
+#### Procedure
+
+1. On Windows and macOS: in a terminal, verify that at least one Podman machine is running:
+
+   ```shell-session
+   $ podman machine list
+   ```
+
+1. To verify you can connect by using the CLI, in a terminal, run the `hello` container:
```

Comment:
```suggestion
1. To verify you can connect using the CLI, in a terminal, run the `hello` container:
```

Reply:
`hello` seems weird as it's not a CLI command, maybe just say 'hello' (with quotes, not `)

Reply:
by using is correct I think.

Reply:
Monospace is not only for CLI commands, but is also used to differentiate user input, which more or less is the case here (that's an image name). Terms enclosed in single quotes is not a thing. Placeholders would be between <angle brackets> and in italic, but that's not the case here.